apps/emqx_connector/src/emqx_connector_resource.erl

```diff
@@ -41,10 +42,10 @@
     reset_metrics/1,
```

Comment:
I think the metrics that was previously in the bridge will now go to the bridge_v2. We might want to have some metrics in the connector but I'm not sure what. So I think we can remove metrics from the connector and maybe add it later if we want it.